### PR TITLE
docs: update the Support policy and schedule

### LINKED ISSUE #35359: Status and Active Date are inconsistent for ^8.0.0 Release
# 📚 Docs or angular.io bug report

### Description

 
^8.0.0 Release is in Active Status, however it's active support ended in Nov 28, 2019. Which should be trusted? The Date or Status?

### What's the affected URL?**
 https://angular.io/guide/releases

### Expected vs Actual Behavior**

Release data to be consisted.

## 📷Screenshot

image

### PR DESCRIPTION
**Pull request by sacgrover**
PR Close #35359

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: 35359

## What is the new behavior?
Information updated

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by kapunahelewong** [comment]
Thanks for this, @sacgrover! We'll just need engineering to review, otherwise it looks good to me!

**Comment by sacgrover** [comment]
@IgorMinar Could you please review this?

File: aio/content/guide/releases.md [hunk 1]
```diff
@@ -105,7 +105,8 @@ The following table provides the status for Angular versions under support.
 
 Version | Status | Released     | Active Ends  | LTS Ends
 ------- | ------ | ------------ | ------------ | ------------
-^8.0.0  | Active | May 28, 2019 | Nov 28, 2019 | Nov 28, 2020
+^9.0.0  | Active | Feb 07, 2020 | Aug 07, 2020 | Aug 07, 2021
```
Comment (kara) [inline aio/content/guide/releases.md]: I believe we released Feb 6

```suggestion
^9.0.0  | Active | Feb 06, 2020 | Aug 06, 2020 | Aug 06, 2021
```

**Review by kara — CHANGES_REQUESTED** [review]
Thanks! I think @StephenFluin should probably review too

File: aio/content/guide/releases.md (hunk 1 above)
Comment (sacgrover) [reply aio/content/guide/releases.md]: OK I will change it and squash the commit.

**Comment by sacgrover** [comment]
@kara  asked changes done

**Comment by sacgrover** [comment]
@kapunahelewong any update on this?

**Review by kara — APPROVED** [review]
LGTM

**Comment by kapunahelewong** [comment]
Hey @sacgrover, it looks good to me, but I have to defer to @kara and @StephenFluin on this one.

**Review by StephenFluin — APPROVED** [review]
This change looks good to me. I also submitted https://github.com/angular/angular/pull/36170/files, but I can rebase that on this change if this one lands first.

**Comment by sacgrover** [comment]
@kapunahelewong Rebased. Thanks :)

**Comment by kapunahelewong** [comment]
Removing Igor because Kara approved. Thanks, all!

**Comment by kapunahelewong** [comment]
Thanks so much, @sacgrover! This is super helpful!

**Comment by sacgrover** [comment]
@kapunahelewong I am glad to be part of it :)

**Pull request closed without merging**